events/blocks/event-mailinglist-form/event-mailinglist-form.css

```diff
@@ -0,0 +1,163 @@
+@media (min-width: 600px) and (max-width: 1200px) {
+  .event-mailinglist-form {
+      width: 675px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .event-mailinglist-form {
+      width: 580px;
+  }
+}
+
+@media (max-width: 599px) {
+  .event-mailinglist-form {
+      width: 91%;
+  }
+}
+
+picture {
+  display: flex;
+}
+
+.mailinglist-form {
+  margin-top: 20px;
+  padding: 72px 36px 0px 36px;
+}
+
+.mailinglist-title {
+  padding-bottom: .5rem;
+  font-size: 2.625rem;
+  color: rgb(44, 44, 44);
+  box-sizing: border-box;
+  font-family: adobe-clean, Source Sans Pro, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, sans-serif;
+  font-weight: 400;
+  line-height: 1.25;
+  margin : 0px;
+}
+
+.mailinglist-description {
+  margin: 0px;
+  padding-bottom: 2rem;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  color: rgb(44, 44, 44);
+}
+
+
+#mailinglist-form {
+  display: table;
+  border-collapse: separate;
+  border-spacing: 0 24px;
+  margin: -20px 0;
+}
+
+.mailinglist-consent-notice {
+  margin-bottom: 8px;
+  margin-top: 0px;
+  padding: 0px;
+  color : #505050;
+  font-size: .75rem;
+  line-height: .875rem;
+  font-weight: 300;
+  margin: 1rem 0 -.5rem;
+
+}
+
+.mailinglist-consent-notice a {
+  color: #2c2c2c;
+  font-weight: 400;
+  text-decoration: underline;
+}
+
+.hide {
+  display: none !important;
```

Comment:
Avoid !important as much as you can. Use specificity level and declaration order to your advantage.